### DISCUSSION TITLE
Problem  with installing Talos on Proxmox

### DISCUSSION THREAD

**Discussion by ebi-droid** [siderolabs/talos#6541] [Q&A] [answered]
talosctl apply-config  -e  10.10.10.31 -n 10.10.10.31 --insecure --file .talos/join.yaml
error applying new configuration: rpc error: code = InvalidArgument desc = configuration validation failed: 1 error occurred:
	* specified install disk does not exist: "/dev/sda"

**Comment by steverfrancis** [comment]
You could try 
`talosctl apply-config -e 10.10.10.31 -n 10.10.10.31 --insecure --mode=interactive`
and find the appropriate disk, then modify the join.yaml file to reflect that...

**Reply by smira** [reply]
you can simply do `talosctl -e 10.10.10.31 disks --insecure` to see the list of blockdevices

**Comment by ebi-droid** [comment, marked as answer]
[SOLVED]
My  nodes were set up on proxmox  so block device where talos is to be installed must be /dev/vda not /dev/sda as in default talos init.yaml config file.
